domain/password.py

```diff
@@ -13,28 +14,29 @@ def random_password():
     return bcrypt.hashpw(random_token(length=12).encode('utf-8'), bcrypt.gensalt())
 
 
-def check_new_password_validity(user, old_password, new_password):
-    errors = ApiErrors()
 
-    if not user.checkPassword(old_password):
-        errors.add_error('oldPassword', 'Ton ancien mot de passe est incorrect.')
-        raise errors
+def check_password_validity(new_password: str, new_confirmation_password: str, old_password: str, user: UserSQLEntity) -> None:
+    api_errors = ApiErrors()
+    _compute_password_strength_errors('newPassword', new_password, api_errors)
```

Comment:
```suggestion
    _check_password_strength('newPassword', new_password, api_errors)
```
remplacer compute par check

Reply:
nous avons préféré 'ensure' que 'check' car ces fonctions ont l'habitude de soulever une exception.